src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -52,6 +53,11 @@
     private $updatePage;
 
     /**
+     * @var HistoryPageInterface
+     */
+    private $historyPage;
```

Comment:
It is just a thought but maybe it would be a cool idea to create `ManagingOrdersHistoryContext` WDYT?